bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -101,25 +102,45 @@ final class BagsManager(executor: ScheduledThreadPoolExecutor) {
       synchronized {
         val key = obs.getNodeKey
         state += key
-        executor.schedule(new Thread {
-          setPriority(Thread.NORM_PRIORITY - 1)
 
-          override def run(): Unit =
+        def bagsStatus(status: BagsStatus) = bagsStatuses.synchronized {
+          val oldState = bagsStatuses.get(key)
+          bagsStatuses += ((key, status))
+          notifyBagsStatusListeners(observation, oldState, Some(status))
+        }
+        def bagsClearStatus() = bagsStatuses.synchronized {
+          val oldState = bagsStatuses.get(key)
+          bagsStatuses -= key
+          notifyBagsStatusListeners(observation, oldState, None)
+        }
+
```

Comment:
I suspect that this has to do with querying the `BagsManager` for the `BagsStatus` as is done in the `GuidingFeedbackEditor`. There may be a way around this by having the `GuidingFeedbackEditor` handle the BAGS status events instead of the `EdCompTargetList`, but this seems a trifle complicated and inconvenient, and will probably require no longer having this class implement the `TelescopePosEditor` as the `edit` method will no longer be sufficient to contain enough information, I wouldn't think, to express this.